.github/ISSUE_TEMPLATE/release-team-lead-progress.md

```diff
@@ -0,0 +1,241 @@
+- [ ] The Communications Lead contacts CNCF to gauge media interest, schedule the CNCF Kubernetes release workshop, and publish the release blog post. Stay in the loop for that. If there is media interest in the release, an interview between the journalist will be organized by the CNCF.
+- [ ] The Release Team Retro part 2 and part 3 is scheduled shortly after the "Release Halfway Point" and a host / facilitator is available
+- [ ] Remind the release team to add items to the retro meeting agenda
+
+### 7. Release Day (~Week 15)
+
+- Note that release day can and should be postponed if any of the conditions outlined in week 11 are not satisfied.
+- Every issue in the milestone is considered release blocking
+- If you have to push the release date back, try to avoid Friday since it makes release publicity extremely difficult. Also, people seem to have patience with delay as long as the reasons are clear and openly communicated. This is your duty. You must over-communicate and ensure the team is also talking to their stakeholders (CNCF, community, press, etc.)
+
+- The following final actions **must occur in order**, with successful completion of each being the entry criteria to the next.
+  - Release day morning:
+    - [ ] Go / No-Go: should generally be clear a day or three ahead of release, but the day's burndown provides a final opportunity for the team to affirm things are ready.
+    - [ ] Release Notes Lead PRs final draft release notes to sig-release, with Release Team Lead approving merge.
+    - [ ] Branch Manager does mock release build.
+    - [ ] Branch Manager does mock publication. Validates with Release Team lead and broader team the mock announcement email content.
+    - [ ] Branch Manager does nomock release build.
+  - Starting when ready:
+    - [ ] Communications Lead begins staging blog post.
+    - [ ] Branch Manager Lead does nomock publication.
+    - [ ] Branch Manager coordinates the building and publishing of rpm/deb packages with Google Build Admins.
+    - [ ] Build Admins affirms build is complete.
+    - [ ] Docs Lead publishes release docs to website.
+    - [ ] Branch Manager does release-notify.
+  - [ ] Approximately 5pm Pacific: all work is complete and the release team
+    lead announces release to [k-dev](https://groups.google.com/a/kubernetes.io/g/dev), [SIG Leads](https://groups.google.com/a/kubernetes.io/g/leads), and discuss.k8s.io.
+- [ ] After the release, it’s time to lift code freeze. The bot will need to be updated.
+  - [Thaw k/k](https://github.com/kubernetes/sig-release/blob/de9e0a9821c4f73b83b4382df7a7dc837f91e18f/releases/release_phases.md#thaw), 
+    master branch is then opened for new pull requests.
+- [ ] The Docs lead thaws k/website
+- [ ] Use all of the appropriate communications channels to announce the lifting of Code Freeze and thawing of 
+  k/website, this can part of the release announcement.
```

Comment:
Does anyone check that the website, release blog, download page etc look right? We could have a checkbox for that.

Reply:
agree, that is a good idea.